apps/ryal_core/web/plugs/json_render_plug.ex

```diff
@@ -8,7 +8,7 @@ defmodule Ryal.JsonRenderPlug do
   alias Ecto.Queryable
   alias Phoenix.Controller
   alias Plug.Conn
-  alias Ryal.Repo
+  alias Dummy.Repo, as: Repo
```

Comment:
I don't like this. I need access to the Repo, so this is what I used. I wish I could do:

```elixir
alias Application.get_env(:ryal_core, :repo), as: Repo
```

Yet it appears I'm not allowed to do that. Thoughts?